src/main/groovy/com/jetbrains/python/envs/PythonEnvsPlugin.groovy

```diff
@@ -83,51 +83,118 @@ class PythonEnvsPlugin implements Plugin<Project> {
                     }
                 }
 
-                condaInstall(project, null, condaExecutable, envs.basePackages)
-//            doFirst {
-//                if (!myExt.bootstrapDirectory.exists()) {
-//                    myExt.bootstrapDirectory.mkdir()
-//                }
-//            }
+                condaInstall(project, condaExecutable, envs.condaBasePackages)
             }
         }
     }
 
-    def createBootstrapJython(project, jythonBootstrapDir) {
-        project.tasks.create(name: 'bootstrapJython') {
-            def conf = project.configurations.jython
+    private void installPythonBuild(Project project, File installDir) {
```

Comment:
Naming is different: the previous one is called createBootstrapConda and this one is InstallPythonBuild, but it creates the task, that installs python build. So either it should be named createInstallPythonBuildTask or previous one should be called bootstrapConda.